recipes/zstd/all/conanfile.py

```diff
@@ -43,9 +43,24 @@ def configure(self):
     def layout(self):
         cmake_layout(self, src_folder="src")
 
+    def _cmake_new_enough(self, required_version):
+        try:
+            import re
+            from io import StringIO
+            output = StringIO()
+            self.run("cmake --version", output=output)
```

Comment:
Would it make sense for conan.tools.cmake.CMake wrapper to have a "version()" ?

@prince-chrismc

Reply:
No :) the graph should be more deterministic regardless of the user's tools.

This version check will go away with new rules in the near future...

The team wants the portable binaries from kitware so it works regardless and doesn't need to be built (which are the reasons this is here)